shared/css/sidebar.css

```diff
@@ -3,96 +3,572 @@
+.media-layout.showing-local-streams > .media-control-buttons > .btn-mute-audio,
+.media-layout.showing-local-streams > .media-control-buttons > .btn-mute-video {
   background-color: inherit;
 }
 
-.media-wrapper.showing-local-streams > .media-control-buttons > .btn-mute-audio:hover:after,
-.media-wrapper.showing-local-streams > .media-control-buttons > .btn-mute-audio:active:after {
-  content: url("/shared/img/audio.svg");
+.media-layout.showing-local-streams > .media-control-buttons > .btn-mute-audio:hover:after,
+.media-layout.showing-local-streams > .media-control-buttons > .btn-mute-audio:active:after {
+  content: url("../img/audio.svg");
 }
 
-.media-wrapper.showing-local-streams > .media-control-buttons > .btn-mute-audio.muted:after {
-  content: url("/shared/img/audio-muted-darkgrey.svg");
+.media-layout.showing-local-streams > .media-control-buttons > .btn-mute-audio.muted:after {
+  content: url("../img/audio-muted-darkgrey.svg");
 }
 
-.media-wrapper.showing-local-streams > .media-control-buttons > .btn-mute-video:after {
+.media-layout.showing-local-streams > .media-control-buttons > .btn-mute-video:after {
   margin-left: -4px;
 }
 
-.media-wrapper.showing-local-streams > .media-control-buttons > .btn-mute-video:hover:after,
-.media-wrapper.showing-local-streams > .media-control-buttons > .btn-mute-video:active:after {
-  content: url("/shared/img/video.svg");
+.media-layout.showing-local-streams > .media-control-buttons > .btn-mute-video:hover:after,
+.media-layout.showing-local-streams > .media-control-buttons > .btn-mute-video:active:after {
+  content: url("../img/video.svg");
 }
 
-.media-wrapper.showing-local-streams > .media-control-buttons > .btn-mute-video.muted:after {
-  content: url("/shared/img/video-muted-darkgrey.svg");
+.media-layout.showing-local-streams > .media-control-buttons > .btn-mute-video.muted:after {
+  content: url("../img/video-muted-darkgrey.svg");
 }
 
-.media-wrapper.showing-local-streams > .media-control-buttons > .general-support-url {
+.media-layout.showing-local-streams > .media-control-buttons > .general-support-url {
   align-items: center;
   display: flex;
   justify-content: center;
 }
 
 /* Highlight on hover. */
-.media-wrapper.showing-local-streams > .media-control-buttons > .general-support-url:hover,
-.media-wrapper.showing-local-streams > .media-control-buttons > .btn:not(.btn-hangup):hover {
+.media-layout.showing-local-streams > .media-control-buttons > .general-support-url:hover,
+.media-layout.showing-local-streams > .media-control-buttons > .btn:not(.btn-hangup):hover {
   background-color: rgba(51, 51, 51, 0.1);
 }
 
-.media-wrapper.showing-local-streams > .remote  {
+/*------------------------------------*\
+    TEXT CHAT
+\*------------------------------------*/
+
+.text-chat-view {
+  flex: 0 0 auto;
+  /* Leaves enough room for the streams and the media buttons */
+  height: calc(100% - 228px);
+  /* Text chat is a fixed 272px width for normal displays. */
+  width: 250px;
```

Comment:
The value and comment are different sizes... maybe just drop the size in the comment?